docs/onboarding_toolkit.md

```diff
@@ -1 +1,172 @@
+* Dedicated communications channels (chat and forum) to communicate with your
+community and attract other community members to your project.
+
+
+
+# Setting up your GitHub repository
+
+The Ansible ecosystem projects exist in the following two GitHub repositories:
+
+* ansible - Access is controlled by Red Hat
+* ansible-community - Access is controlled by the community, 
+under Red Hat sponsorship
+
+We recommend most Ansible projects use the ansible-community GitHub repository.
+Repository transfers are available if your project already exists somewhere else in GitHub.
+
+For new projects, use the [project-template](https://github.com/ansible-community/project-template) to 
+create a project with the default files.
+
+## Other repository settings
+
+* Open up issues and PRs for task tracking and community contributions.
+* Disable the GitHub discussions. Use the 
+[Ansible Community forum](https://forum.ansible.com/) for community and project discussions.
+* Disable the repo wiki for project documentation. See the sections below on 
+documentation and using the Ansible forum.
+
+## Common files
+
+*Note:* If you use the Ansible GitHub project template, these files are included
+in the created repository.
+
+All Ansible ecosystem projects MUST include the following files:
+
+* [COPYING](https://github.com/ansible-community/onboarding/blob/main/COPYING) - 
+If your project does not already have a license, we recommend
+[GPLv3](https://www.gnu.org/licenses/gpl-3.0.txt).
+* [CONTRIBUTING](https://github.com/ansible-community/onboarding/blob/main/CONTRIBUTING.md) - 
+This can be the complete contributor details, if it is short. 
+Otherwise, it can point to your project documentation which should include
+ a Contributor guide. It also should include your Ansible Matrix chat channels and 
+ [Ansible Community Forum](https://forum.ansible.com/) tags.
+* [Code of Conduct](https://github.com/ansible-community/onboarding/blob/main/CODE-OF-CONDUCT.md) - 
+MUST link to the [Ansible code of conduct](https://docs.ansible.com/ansible/devel/community/code_of_conduct.html). 
+    Do not copy the CoC.
+[DCO](https://github.com/ansible-community/onboarding/blob/main/DCO) - The Developer 
+Certificate of Origin. Do not modify this text.
+* [README.md](https://github.com/ansible-community/onboarding/blob/main/README.md) - 
+A basic introduction to your project, communication channels, and so on.
+It should include your matrix channel if your project has one, and  recommended forum tags,
+with a link to the [Ansible Community Forum](https://forum.ansible.com/). 
+You should not use the README.md for project documentation (user guide, for example),
+ but instead set up a docsite. Even if the docsite is basic, it's a start others can
+contribute to.
+
+## Optional other steps
+While not required, you should consider adding the following to your project repository:
+
+  * Add a [SECURITY.md](https://github.com/ansible-community/onboarding/blob/main/SECURITY.md) 
```

Comment:
I don't think this should be optional.
If the repo is part of Ansible's GitHub Org then we should direct to `security@ansible.com`